website/docs/faqs/Troubleshooting/job-memory-limits.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "I'm receiving a 'This run exceeded your account's run memory limits' error in my failed job"
+description: "Use incremental models or optimize queries for job failures due to exceeded memory limits."
+sidebar_label: 'Job failures due to exceeded memory limits'
+---
+
+If you're receiving a `This run exceeded your account's run memory limits` error in your failed job, it means that the job exceeded the memory limits set for your account. Memory limits are on a per run basis and are typically influenced by the amount of result data that dbt has to ingest and process, which typically is quite small but can be bloated unexpectedly by project design choices.
+
+## Reasons
```

Comment:
maybe this is better?
```suggestion
## Common reasons
```

Reply:
Hiya @mirnawong1 

I quite like **Common reasons** as a title for the section.
**Common reasons** indicate these are the reasons we've come across for the job error and the steps in the doc might help. Whereas **Reasons** indicates THIS IS / THESE ARE the reasons which kinda goes against the section on reaching out to support if the troubleshooting steps don't work.
Just my two cents.

Kind Regards
Natalie